### PR TITLE
accomplish multiLanguage design

### PR DESCRIPTION
**Pull request by ramsayleung**
1. refactor Category's relationship with other table from oneToMany to ElementCollection and CollectionTable.
2. refactor LocalizedStringValue from Entity to Embeddable

### CODE REVIEW

**Review by davis999 — COMMENTED** [review]
1. remove useless comments.
2. have tested?

**Comment by ramsayleung** [comment]
I have run rest-test after refactoring code.

**Comment by ramsayleung** [comment]
1. delete useless comments.
2. run rest-test after refactoring code, all tests pass

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -24,4 +24,8 @@ io:
       brokerType: google
       google:
         project:
-          id: reactivesw-project
\ No newline at end of file
+          id: reactivesw-project
+      topic:
+        deletecategory:
+          name: "reactivesw-dev-category-deleted"
+          version: 1
```
Comment (davis999) [inline src/main/resources/application.yml:31]: should merge from dev.

File: src/main/java/io/reactivesw/category/domain/model/Category.java [hunk 2]
```diff
@@ -34,6 +32,7 @@
 @EntityListeners(AuditingEntityListener.class)
 public class Category {
 
+
```
Comment (davis999) [inline src/main/java/io/reactivesw/category/domain/model/Category.java:35]: why add a blank line?

**Pull request closed without merging**